### PR TITLE
feat(e2e): add removable card e2e test to experiences

### PR DESCRIPTION
**Pull request by davidmohringmrf**

### CODE REVIEW

File: e2e/utils/bootstrap.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ const bootstrapExperience = async(browser, config, fixture) => {
 		() => async() => { //depends on selected article
 			return await browser.getTitle() === fixture.articleTitle;
 		},
-		{ timeout: 10000, interval: 1000 }
+		{ timeout: 10000, interval: 1000, timeoutMsg: `${fixture.articleTitle} didnt load correctly` }
```
Comment (oscard0m) [inline e2e/utils/bootstrap.js]: **[💅🏽 nit-pick]**
```suggestion
		{ timeout: 10000, interval: 1000, timeoutMsg: `${fixture.articleTitle} didn't load correctly` }
```

File: e2e/utils/card.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+	await browser.waitUntil(async()=>{
+		isExisting = await card.isExisting();
+
+		await scrollBy(browser, 20);
+
+		return isExisting;
+	}, {
+		timeout: 10000,
+		interval: 1000,
```
Comment (oscard0m) [inline e2e/utils/card.js:20]: Could this interval be one of the reasons why are we slow?

File: e2e/utils/card.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+		timeoutMsg: 'Card didn\'t exist'
+	});
+
+	return isExisting;
+};
```
Comment (oscard0m) [inline e2e/utils/card.js]: **[💅🏽 nit-pick]** I would have wrapped it in a function `scrollUntilCardAppears` or something similar

File: e2e/utils/card-actions/remove.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+const { dragCardTo } = require('../touch');
+
+const removeCard = async(browser, cardSelector) => {
+	const height = await browser.executeAsync(async(args, done) => {
+		const windowHeight = window.screen.height;
+
+		done(windowHeight);
+	}, '');
```
Comment (oscard0m) [inline e2e/utils/card-actions/remove.js:8]: **[💅🏽  nit-pick]** Maybe this utility is useful for this use case? https://webdriver.io/docs/api/browser/getWindowSize/

File: e2e/utils/touch.js [hunk 5]
```diff
@@ -1,10 +1,40 @@
...
+		}
+	} else {
+		while (localFrom+step < to) {
+			ar[ar.length] = localFrom+= step;
+		}
+	}
+	ar[ar.length] = to;
+
+	return ar;
+};
```
Comment (oscard0m) [inline e2e/utils/touch.js]: **[💅🏽  nit-pick]**

I think this kind of utilities is worth to consider an external library doing this for us. I found this lib which seems to have a good adoption. Would its range utility be useful for our use case?
https://mathjs.org/docs/reference/functions/range.html

**Review by oscard0m — DISMISSED** [review]

File: e2e/utils/card-actions/remove.js (hunk 4 above)
Comment (davidmohringmrf) [reply e2e/utils/card-actions/remove.js:8]: I tried, it doesn't work for android :(

File: e2e/utils/touch.js (hunk 5 above)
Comment (davidmohringmrf) [reply e2e/utils/touch.js]: Oh, yeah, now that we have it in our nodejs scope, we can use a library, I did this to be able to execute it inside the browser, but the refactor placed it outside, I'll add the library

File: e2e/utils/card.js (hunk 2 above)
Comment (davidmohringmrf) [reply e2e/utils/card.js:20]: That the card exists is critical for all the tests, so we have to wait the time needed for it to appear. In most cases it appear immediately.

**Review by oscard0m — DISMISSED** [review]

File: e2e/utils/card.js (hunk 2 above)
Comment (oscard0m) [reply e2e/utils/card.js:20]: > That the card exists is critical for all the tests, so we have to wait the time needed for it to appear. In most cases it appear immediately.

I was thinking more in terms if 1000 is too much or not

**Review by oscard0m — APPROVED** [review]

**Review by davidramosmrf — COMMENTED** [review]
LGTM

**Review by enricgironamrf — APPROVED** [review]

**Pull request merged**